### PR TITLE
Treat broadcaster as mod

### LINKED ISSUE #9: Streamer should count as a mod
It doesn't seem like the streamer counts as a 'mod' for purpose of 'mod-only commands', meaning streamer can't easily create/delete their own commands.

Example debug output, showing the user-type being empty:
> 2017/07/24 16:52:36 [IN]  @badges=moderator/1,subscriber/0,turbo/1;color=#5F5775;display-name=r4v5;emotes=433:65-66;id=156a2dbc-ff2f-404e-99fb-4abc589a2204;mod=1;room-id=113481237;subscriber=1;tmi-sent-ts=1500933155182;turbo=1;user-id=39399416;user-type=mod :r4v5!r4v5@r4v5.tmi.twitch.tv PRIVMSG #bengineering :Yep, still need to fix that so that the streamer counts as a mod
> 2017/07/24 16:52:51 [IN]  @badges=broadcaster/1,subscriber/0;color=#DAA520;display-name=Bengineering;emotes=;id=c25b5f41-5884-48ff-b638-b9a58baba0ee;mod=0;room-id=113481237;sent-ts=1500933126760;subscriber=1;tmi-sent-ts=1500933170337;turbo=0;user-id=113481237;user-type= :bengineering!bengineering@bengineering.tmi.twitch.tv PRIVMSG #bengineering :ohhhh alright cool I thought I was crazy

### PR DESCRIPTION
**Pull request by rakslice**
Fixes #9

### CODE REVIEW

**Review by FugiTech — CHANGES_REQUESTED** [review]
I like the idea of implicitly treating the broadcaster as a mod but...

File: handler.go [hunk 1]
```diff
@@ -103,7 +103,8 @@ func handle(out chan string, m *message) {
 		for _, prefix := range cmdPrefixes {
 			if strings.HasPrefix(msg, prefix) {
 				p := split(m.Args[1][len(prefix):], 2)
-				if c := cmds.Get(p[0]); c != nil && (!c.modOnly || m.Mod) {
+				isMod := m.Mod || strings.ToLower(m.DisplayName) == CHANNEL
```
Comment (FugiTech) [inline handler.go]: Display names can differ from login for Asian users as well as special channels like Riot Games. The easiest safe way to check afaik is to see if the "user-id" and "room-id" tags match

**Comment by rakslice** [comment]
How's this?

**Review by FugiTech — CHANGES_REQUESTED** [review]

File: message.go [hunk 2]
```diff
@@ -11,6 +11,8 @@ type message struct {
 	Mod         bool
 	Sub         bool
 	Command     string
+	RoomId      string
```
Comment (FugiTech) [inline message.go]: In Go it should always be `id` or `ID`, never `Id`

Comment (rakslice) [reply message.go]: OIC. https://github.com/golang/go/wiki/CodeReviewComments#initialisms

**Comment by rakslice** [comment]
Ok...

**Pull request merged**